turbinia/turbiniactl.py

```diff
@@ -818,21 +815,20 @@ def main():
               '{1!s}: {2!s}'.format(pair, args.recipe_config, exception))
           sys.exit(1)
         request.recipe[key] = value
-    if args.dump_json:
-      print(request.to_json().encode('utf-8'))
-      sys.exit(0)
     else:
```

Comment:
I think you want to remove this `else` statement as well.  Since you remove the previous `if`, this now pairs with the `if args.recipe_config`.

Reply:
good catch, done.